gulpfile.js

```diff
@@ -103,8 +103,11 @@ gulp.task('copy:main.css', function () {
 
     return gulp.src(dirs.src + '/css/main.css')
                .pipe(plugins.header(banner))
-               .pipe(gulp.dest(dirs.dist + '/css'));
-
+                .pipe(plugins.autoprefixer({
+                    browsers: ['last 2 versions', 'ie >= 8'],
```

Comment:
@AllThingsSmitty: sorry for not being clear enough. I'm well aware of how Autoprefixer works, that's not the issue
I'm trying to point out. My concern is if the `last 2 versions` are the accurate number of previous versions that H5BP
officially supports —the docs only mention "Chrome, Firefox, IE8+, Opera, Safari".

With the current config —`['last 2 versions', 'ie >= 8']`—, we'll get:

```css
@media print,
       (-webkit-min-device-pixel-ratio: 1.25),
       (min-resolution: 120dpi) {
    /* Style adjustments for high resolution devices */
}
```

Instead of the current*:

```css
@media print,
       (-o-min-device-pixel-ratio: 5/4),
       (-webkit-min-device-pixel-ratio: 1.25),
       (min-resolution: 120dpi) {
    /* Style adjustments for high resolution devices */
}
```

*To achieve the current one, we need to add at least `opera >= 12`*.

Besides that, my main concern is how many previous versions of each browser H5BP supports.